demo/test_inbox.py

```diff
@@ -64,13 +64,19 @@ def make_user(**kwargs):
     return models.User(**defaults)
 
 
+def now():
+    # Workaround for Windows where two close call to datetime.now() return
+    # exaclty the same datetime
+    return datetime.now() + timedelta(microseconds=last_auto_id)
```

Comment:
Please guard this with a conditional so that the workaround is only applied when executing on a windows platform.